Dockerfile

```diff
@@ -170,6 +174,9 @@ COPY --from=charts \
     /charts/ \
     /charts/
 
+FROM scratch AS runtime
+COPY --from=runtime-collect / /
```

Comment:
Flattening all this into a single layer makes the runtime extraction a couple seconds faster